src/libraries/pkg/Microsoft.NETCore.Platforms/runtimeGroups.props

```diff
@@ -118,16 +118,10 @@
       <TreatVersionsAsCompatible>false</TreatVersionsAsCompatible>
     </RuntimeGroup>
 
-    <RuntimeGroup Include="osx">
-      <Parent>unix</Parent>
-      <Architectures>x64</Architectures>
-      <Versions>10.10;10.11;10.12;10.13;10.14;10.15</Versions>
-    </RuntimeGroup>
-
     <RuntimeGroup Include="osx">
       <Parent>unix</Parent>
       <Architectures>x64;arm64</Architectures>
-      <Versions>11;11.0</Versions>
+      <Versions>10.10;10.11;10.12;10.13;10.14;10.15;11.0</Versions>
```

Comment:
Were we going to add 10.16 here too, or are we certain we won't need it?

Reply:
I added it as well since we did that for 2.1 and 3.1. I think we should need it now since we _should_ report proper 11.0 version but I think it will make it easier to maintain if this is same in all supported branches.